woocommerce/woocommerce-gutenberg-products-block
# Product Ratings: Add Global Styles font size and spacing support

This PR adds Global Styles font size and spacing support to the Product Rating block.


Fixes #5669 

### Screenshots
|Before|After|
|-|-|
|![Product_Ratings_Block_–_kirigami](https://user-images.githubusercontent.com/905781/155368942-e2fee34d-2d15-403b-a9eb-b09c001d7384.jpg)|![Product_Ratings_Block_–_kirigami](https://user-images.githubusercontent.com/905781/155367945-af959459-28f1-4b00-897f-fd5a90b77d43.jpg)|

### Manual Testing
How to test the changes in this Pull Request:

1. Check out this branch and compile the changes
2. Activate a **block theme**, like Twenty Twenty Two
3. Create a new page, and add the **All Products** block
4. Edit the **Product Ratings** block, and make sure the **Font size** and **Spacing** block settings are present
5. Ensure the **Font size** and **Spacing** customizations work correctly both on the block level and on the Global Styles level

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).


Co-authored-by: gigitux <gigitux@gmail.com>

#### Automated Tests

### Changelog

> Product Ratings: Add Global Styles font size and spacing support
